pom.xml

```diff
@@ -5,20 +5,78 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>dev.cdevents</groupId>
 	<artifactId>cdevents-sdk-java</artifactId>
-	<version>0.0.1</version>
-	<packaging>jar</packaging>
+	<version>0.0.1-SNAPSHOT</version>
+
 	<name>cdevents-sdk-java</name>
 	<description>cdevents java sdk</description>
+	<url>https://github.com/cdevents</url>
+	<inceptionYear>2022</inceptionYear>
+
+	<organization>
+		<name>Continuous Delivery Foundation</name>
+		<url>https://cd.foundation</url>
+	</organization>
 
 	<properties>
+		<project.identifier>${project.name}</project.identifier>
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<cloudevents.version>2.3.0</cloudevents.version>
 		<junit-jupiter.version>5.8.2</junit-jupiter.version>
-		<assertj-core.version>3.22.0</assertj-core.version>
-		<jackson.version>2.13.2</jackson.version>
+        <assertj-core.version>3.22.0</assertj-core.version>
+        <jackson.version>2.13.2</jackson.version>

```

Comment:
I'm not sure what these jackson properties are 😅 sorry about my ignorance on this! 
Are these versions generated somehow or maintained by hand?

Reply:
These expressions are written by hand, typically used to refactor common values.